src/Sylius/Behat/Page/Admin/Crud/EditPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Crud;
+
+use Behat\Mink\Session;
+use Sylius\Behat\Page\SymfonyPage;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class EditPage extends SymfonyPage implements EditPageInterface
+{
+    /**
+     * @var string
+     */
+    private $resourceName;
```

Comment:
Because it would change our convention. If we add this getter and make property private we should do the same with rest of `resourceName` in this PR. If we do so, we should change accessor for `tableManipulator` too. And at the end, few pages would have different API than others. It is not good to change conventions behind the curtain.